package.json

```diff
@@ -38,6 +38,9 @@
     "@types/vinyl": "^1.1.29",
     "@types/vinyl-fs": "0.0.28",
     "@types/yeoman-generator": "0.0.30",
+    "bower": "~1.8.0",
```

Comment:
I'd still like to see us pin to a specific version. I know plenty of big projects that would treat a refactoring of files or private properties as a patch no-breaking-changes bug fix.